build.psm1

```diff
@@ -561,6 +575,8 @@ function Start-PSPester {
     )
 
     Write-Verbose "Running pester tests at '$path' with tag '$($Tag -join ''', ''')' and ExcludeTag '$($ExcludeTag -join ''', ''')'" -Verbose
+    # Publish EchoArgs so it can be run by tests
+    Publish-EchoArgs
```

Comment:
I like the way you structured it! One nit-picking: maybe rename it to `Publish-PSTestTools`, so we can use to for similar cases. And move the comment above to the function.